evalml/automl/engine/cf_engine.py

```diff
@@ -135,14 +132,16 @@ def submit_scoring_job(
     ) -> EngineComputation:
         """Send scoring job to cluster.
 
-        Arguments:
-            automl_config: structure containing data passed from AutoMLSearch instance
-            pipeline (pipeline.PipelineBase): pipeline to train
-            X (pd.DataFrame): input data for modeling
-            y (pd.Series): target data for modeling
-        Return:
-            CFComputation: a object wrapping a reference to a future-like computation
-                occurring in the resource pool
+        Args:
+            automl_config: Structure containing data passed from AutoMLSearch instance.
+            pipeline (pipeline.PipelineBase): Pipeline to train.
+            X (pd.DataFrame): Input data for modeling.
+            y (pd.Series): Target data for modeling.
+            objectives (list(ObjectiveBase)): Objectives to score on.
```

Comment:
It caught the missing `objectives` parameter! Seems like sometimes it fails to catch all missing parameters though :/

Reply:
@angela97lin It's this bug: https://github.com/PyCQA/pydocstyle/issues/45 The section needs to be called `Return` not `Returns`. I ran into this a year ago when I first opened the issue - one of the reasons I decided not to go forward with this is that this bug means that there's a risk our docstrings could miss params even with linting - which seemed to defeat the purpose.

Is there a way to enforce the use of `Return` instead of `Returns` ?

Reply:
There was another bug about not enforcing the line break if there weren't periods at the end of the args: https://alteryx.quip.com/6ib5AbEjXB78/Docstring-Formatting#KXTACAmPxNH 

Is that still an issue?

Reply:
@freddyaboulton I see--I don't think so with just `pydocstyle` alone 😬 those are still issues 😭 

I did find this: https://pythonrepo.com/repo/terrencepreilly-darglint-python-linters-style-checkers but haven't gotten a chance to poke around with it yet! Will try to take a peek :')